tracer/build/_build/Build.Steps.cs

```diff
@@ -354,20 +356,27 @@ partial class Build
                     .SetTargetPlatform(platform)));
         });
 
-    Target CompileNativeTestsLinux => _ => _
+    Target CompileTracerNativeTestsLinux => _ => _
         .Unlisted()
         .After(CompileNativeSrc)
         .OnlyWhenStatic(() => IsLinux)
         .Executes(() =>
         {
-            Logger.Error("We don't currently run unit tests on Linux");
+            EnsureExistingDirectory(NativeBuildDirectory);
+
+            CMake.Value(
+                arguments: $"-DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang -B {NativeBuildDirectory} -S {RootDirectory} -DCMAKE_BUILD_TYPE=Release");
+            CMake.Value(
+                arguments: $"--build {NativeBuildDirectory} --parallel {Environment.ProcessorCount} --target {FileNames.NativeTracerTests}");
         });
 
     Target CompileNativeTests => _ => _
         .Unlisted()
         .Description("Compiles the native unit tests (native loader, profiler)")
-        .DependsOn(CompileNativeTestsWindows)
-        .DependsOn(CompileNativeTestsLinux)
+        .DependsOn(CompileTracerNativeTestsWindows)
+        .DependsOn(CompileTracerNativeTestsLinux)
+        .DependsOn(CompileNativeLoaderTestsWindows)
+        .DependsOn(CompileNativeLoaderTestsLinux)
         .DependsOn(CompileProfilerNativeTestsWindows);
```

Comment:
Don't we need 
    `.DependsOn(RunProfilerNativeUnitTestsLinux)`

Reply:
no the target `CompileNativeTests` depends on other compilation not on the run taget